heroic-core/src/main/java/com/spotify/heroic/metric/MetricManagerModule.java

```diff
@@ -88,6 +90,16 @@
      */
     private final boolean failOnLimits;
 
+    /**
+     * If {@code true}, will enable logging of queries
+     */
+    private final boolean logQueries;
```

Comment:
I've done this now, but am wondering how this would be configured to be enabled by default without having to recompile.
I know that Feature flags can be set in the query - can a default also be set in the configuration file?